src/Leaderboard.py

```diff
@@ -188,6 +188,11 @@ def showLeaderboard(player: str = None, limit: int = None) -> str or List[str]:
         return msgs if len(msgs) > 1 else msgs[0]
 
 
+def emptyLeaderboard() -> bool:
```

Comment:
`emptyLeaderboard` sounds like you are emptying/deleting the leaderboard. I would just do `leaderboard.count() < 6` inline instead of making a separate function for it.

Reply:
Resolved.